src/components/attendee/VideoComms/scss/VideoCommsParticipant.module.scss

```diff
@@ -18,26 +20,27 @@
   display: flex;
   flex-direction: row-reverse;
   font-size: 22px;
-  color: #f1f1f1;
+  color: theme.$c-light-text-on-dark;
   background: rgba(0, 0, 0, 0.5);
   border-top-left-radius: 4px;
 
   span {
     margin: 2px 7px;
   }
   &__darkButtons {
-    color: #000000;
+    color: theme.$c-default-text;
     background: none;
     cursor: pointer;
   }
 }
 
 .avatarContainer {
   display: flex;
-  border: 1px solid #dbdbdb;
-  background: #f0f0f0;
+  border: 1px solid theme.$sparkle--border;
```

Comment:
[NOTE]
if there are plenty `1px solid $sparkle--border` and maybe few other common border styles, these can also be moved to the standard library.